schema.json

```diff
@@ -2105,7 +2100,7 @@
       },
       {
         "type": "checkbox",
-        "label": "Show Copyright Info",
+        "label": "Show \"©\", current year and store name",

```

Comment:
@bc-ong this is the line that i was worrying would require some hackery to get a dynamic store and year